### DISCUSSION TITLE
GitHub doesn't properly support Org-mode line breaks

### DISCUSSION THREAD

**Discussion by rilysh** [community/community#75504] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello! I'm using an Org file for one of my README, and I noticed that GitHub doesn't properly support Org's enforced line breaks (`\\`). For the first line, it works but then it prints the line break character itself.

### Screenshots (GitHub)
- Source
![image](https://github.com/community/community/assets/71683721/e98c7100-fbf6-4505-a94a-cec0a878c27d)
- After rendering
![image](https://github.com/community/community/assets/71683721/928e63b5-c8f2-4f3b-ab95-af0dd0ccb5d7)

### Screenshots (Gitea)
- Source
![image](https://github.com/community/community/assets/71683721/5b6f69db-3f77-4792-a980-a5baa6b9216c)
- After rendering
![image](https://github.com/community/community/assets/71683721/3fdf9144-9d2a-496e-bcfd-c703944645b1)

As you can see, I expect the result to be the same as Gitea. Here's the documentation of [paragraph line breaks](https://orgmode.org/org.html#Paragraphs).

**Comment by golanv** [comment]
Yep, I can confirm this issue. You can use something like...
```
#+BEGIN_HTML
a line <br>
a new line <br>
#+END_HTML
```
But, that feels excessive just to get line breaks working properly on GitHub.
